dms/i18n/es.po

```diff
@@ -1173,6 +1173,12 @@ msgstr "Tiene mensaje"
 msgid "Human Resource"
 msgstr "Recursos humanos"
 
+#. module: dms
+#: model:ir.model.fields,field_description:dms.field_dms_directory__human_size
+#: model:ir.model.fields,field_description:dms.field_dms_file__human_size
+msgid "Human Size"
```

Comment:
This should change as well, or remove the changes and let the bot to add them later